plasmapy/classes/plasma.py

```diff
@@ -5,12 +5,19 @@
 import numpy as np
 import astropy.units as u
 
-mu0 = np.pi * 4.0e-7 * (u.newton / (u.amp**2))
+from ..constants import (m_p, m_e, c, mu0, k_B, e, eps0, pi)
```

Comment:
Might as well throw in absolute imports here!

Reply:
done